registry.json

```diff
@@ -117,6 +117,17 @@
     "contributors": []
   },
   {
+    "name": "youtube-deno",
+    "nestdotland": "nest.land/package/youtube-deno",
+    "denoland": "deno.land/x/youtube@v0.3.0",
+    "author": {
+      "name": "Aksh Gupta",
+      "github": "akshgpt7",
+      "wallet": "LvguIXMVoJWLDAw8Cj42qmwrrfYG2U_36iprvXOfKDo
+"
+    },
+    "contributors": []
+  },
```

Comment:
```suggestion
  },
  }
```

Reply:
Sorry about this. This may be an issue that came up when you updated your branch with master (my fault).